src/utils/devices.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Defines "browser" as having the `window` global
+ * Checks if the browser supports them
+ * @return {boolean}
+ */
+function isBrowser() {
+	return (typeof window !== 'undefined');
+}
+
+/**
+ * Is it apple safari browser
+ * @return {boolean}
+ */
+export function isSafari() {
+	return isBrowser() && window.navigator.userAgent.includes('Safari')
```

Comment:
Should we toLowerCase these and then do `safari`?

Reply:
Ended up making it a prop so we can control the logic in website and keep Maker browser agnostic